source/contracts/reporting/IMarket.sol

```diff
@@ -38,13 +38,20 @@ contract IMarket is Typed, IOwnable {
     function getShareToken(uint8 _outcome)  public constant returns (IShareToken);
     function getDesignatedReporterDisputeBondToken() public constant returns (IDisputeBond);
     function getLimitedReportersDisputeBondToken() public constant returns (IDisputeBond);
+    function getAllReportersDisputeBondToken() public constant returns (IDisputeBond);
     function getMarketCreatorSettlementFeeInAttoethPerEth() public constant returns (uint256);
     function getReportingState() public constant returns (ReportingState);
     function getFinalizationTime() public constant returns (uint256);
     function getFinalPayoutDistributionHash() public constant returns (bytes32);
     function getFinalWinningReportingToken() public constant returns (IReportingToken);
     function getReportingTokenOrZeroByPayoutDistributionHash(bytes32 _payoutDistributionHash) public constant returns (IReportingToken);
-    function getPayoutDistributionHashStake(bytes32 _payoutDistributionHash) public constant returns (int256);
+    function getTentativeWinningPayoutDistributionHash() public constant returns (bytes32);
+    function getBestGuessSecondPlaceTentativeWinningPayoutDistributionHash() public constant returns (bytes32);
+    function getForkingMarket() public constant returns (IMarket _market);
+    function getEndTime() public constant returns (uint256);
+    function getDesignatedReportDueTimestamp() public constant returns (uint256);
+    function getDesignatedReportReceivedTime() public constant returns (uint256);
+    function getDesignatedReportDisputeDueTimestamp() public constant returns (uint256);
```

Comment:
The fact that these weren't in the interface suggests they weren't being called externally.  Instead of adding them to the interface, consider making them private.  The exception to this is getters that are just blindly returning some contract state, which we do want to make available to the general public.  Over time I suspect the set of public methods will grow as the UI asks us for access to things, but I would prefer to start small and expand rather than starting with everything.

Reply:
I added them to the interface so that the `MarketExtensions` contract could call them. This could be optimized out at some point probably by passing this data to those functions, although that will be a battle with stack size limit I'm guessing.